controllers/prep_handlers.go

```diff
@@ -444,6 +444,15 @@ func (r *ImageBasedUpgradeReconciler) handlePrep(ctx context.Context, ibu *lcav1
 
 	switch {
 	case !r.PrepTask.Active:
+		// validate health before activating the worker
```

Comment:
nit: this comment i think can be removed since the log below it says the same same thing.

Reply:
Removed